components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/Util.java

```diff
@@ -198,8 +210,63 @@ public static void updateCheckboxStateMap(Map<String, Boolean> checkBoxMap, Map<
         }
     }
 
-    public static boolean isAskPasswordEnabled(){
+    public static boolean isAskPasswordEnabled() {
         return isAskPasswordEnabled;
     }
 
+    public static boolean isUserOnBoardingEnabled(ServletContext context, HttpSession session) throws Exception {
+        Object emailVerificationEnabledObj = session.getAttribute(EMAIL_VERIFICATION_ENABLE_PROP_NAME);
+        if (emailVerificationEnabledObj != null) {
+            return Boolean.parseBoolean(String.valueOf(emailVerificationEnabledObj));
+        }
+        String backendServerURL = CarbonUIUtil.getServerURL(context, session);
+        String cookie = (String) session.getAttribute(ServerConstants.ADMIN_SERVICE_COOKIE);
+        ConfigurationContext configContext =
+                (ConfigurationContext) context.getAttribute(CarbonConstants.CONFIGURATION_CONTEXT);
+        Map<String, Map<String, List<ConnectorConfig>>> connectorList;
+        try {
+            IdentityGovernanceAdminClient client =
+                    new IdentityGovernanceAdminClient(cookie, backendServerURL, configContext);
+            connectorList = client.getConnectorList();
+        } catch (RemoteException e) {
+            throw new Exception("Error while calling governance service", e);
```

Comment:
Better if the error message
"Error while getting connector list from governance service, at URL :" + backendServerURL